pkg/controller/volume/expand/pvc_populator.go

```diff
@@ -56,7 +56,7 @@ type pvcPopulator struct {
 	recorder        record.EventRecorder
 }
 
-func NewPVCPopulator(
+func newPVCPopulator(
 	loopPeriod time.Duration,
```

Comment:
Actually - this file should be deleted. It was a mistake on my part to leave this file around. Can you do that please?

Reply:
Yes，I will do it